src/reanimated2/Colors.ts

```diff
@@ -7,18 +7,29 @@
  */
 
 /* eslint no-bitwise: 0 */
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-nocheck
 import { Platform } from 'react-native';
 import { makeRemote, makeShareable, isConfigured } from './core';
 import { interpolate } from './interpolation';
+// @ts-ignore JS file
 import { Extrapolate } from '../reanimated1/derived';
 
+interface RBG {
+  r: number;
+  g: number;
+  b: number;
+}
+
+interface HSV {
+  h: number;
+  s: number;
+  v: number;
+}
+
 // var INTEGER = '[-+]?\\d+';
 const NUMBER = '[-+]?\\d*\\.?\\d+';
 const PERCENTAGE = NUMBER + '%';
 
-function call(...args) {
+function call(...args: unknown[]) {
```

Comment:
It seems the return type is missing. `: string`

Reply:
Thanks, I must've missed that.

Reply:
Fixed in e8d1243a.